logconsumer_test.go

```diff
@@ -21,20 +21,32 @@ import (
 const lastMessage = "DONE"
 
 type TestLogConsumer struct {
-	Msgs []string
-	Ack  chan bool
+	Msgs     []string
+	Done     chan bool
+	Accepted chan string
```

Comment:
Could you add a comment explaining why we are adding the Accepted channel here, in combination with the Msgs slice? 🙏